enhancements/ingress/set-delete-http-headers.md

```diff
@@ -1002,16 +1006,27 @@ This route can be annotated with `haproxy.router.openshift.io/disable_cookies=tr
   IngressController.
 
 #### Why does a custom header value defined in RouteSpec override IngressControllerSpec value ?
-  Answer: It's quite common to have a global setting with an escape hatch locally to override the global setting.
-  The purpose of this EP is to define custom headers. So, there are two reasons we need such override:
-  1. A route owner shall have the choice to define what headers he wants as per his application requirements(as they might be varying) 
-  than the ones admin has set cluster-wide.
-  2. The IngressController and Route might be owned by the same person who wants to specify some global configuration
-  and then override that global configuration on a small subset of routes. 
+  Answer: In case of HTTP response headers, the headers set in spec.httpHeaders.actions of IngressController will override
+ the headers with same name in Route's spec.httpHeaders.actions or Route annotation.
+
+ In case of HTTP request headers, the headers set in Route using spec.httpHeaders.actions or annotation will override the 
+ headers with the same name present in the IngressController's spec.httpHeaders.actions.
+
+ This is because of the way Haproxy is designed to work. We cannot change the processing order; we're constrained by HAProxy's design: 
```

Comment:
nit: use the same capitalization for HAProxy throughout.
```suggestion
 This is because of the way HAProxy is designed to work. We cannot change the processing order; we're constrained by HAProxy's design: 
```

Reply:
done